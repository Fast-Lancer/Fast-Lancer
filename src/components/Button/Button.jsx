export default function Button({ handleClick, buttonText }) {
  return <button onClick={handleClick}>{buttonText}</button>
}
