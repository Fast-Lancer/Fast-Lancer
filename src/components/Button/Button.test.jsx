import { render } from '@testing-library/react'
import { BrowserRouter as Router, } from 'react-router-dom'
import Button from './Button'

it('renders the button component accurately', () => {
  const { container } = render(
    <Router>
      <Button buttonText='test'/>
    </Router>
  )
  expect(container).toMatchSnapshot()
})
