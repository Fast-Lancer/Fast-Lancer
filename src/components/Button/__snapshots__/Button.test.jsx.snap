// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the button component accurately 1`] = `
<div>
  <button>
    test
  </button>
</div>
`;
